static/website/templates/index.html

```diff
@@ -3,102 +3,113 @@
+            </div>
+        </div>
+        {% endfor %}
+    </div> <!-- /.carousel -->
 </div> <!-- /#carousel-container -->
 
 <div id="filter-container">
-	<div class="row" style="margin-left: 12%;">
-		<form class ="" id="filter-form" action="" role="form">
-			<div class="col-lg-6 col-md-6 col-sm-6 col-xs-6" style="width: 65%;">
-				<a class="btn btn-block btn-primary" href="{% url 'website:questions' %}" style="margin-top: 18px;">View all previous questions.</a>
-			</div> <!-- /.col -->
-		</form>
-		<div class="col-lg-6 col-md-6 col-sm-6" style="width: 25%;">
-			<div class="row" style="margin-left: 11px;">
-
-				<div class="col-lg-6 col-md-6 col-sm-6 col-xs-6" style="width: 100px; margin-right:40px; " >
-					<h3 style="font-size: 18px" align="center"><strong>{% total_question_count %}</strong></h3>
-					<h3 style="font-size: 18px; margin-top: 1px;" align="center" text-color="#7395d9" >Questions</h3>
-				</div>
-				
-			</div>
-		</div>
-	</div> <!-- /.row -->
+    <div class="row" style="margin-left: 12%;">
+        <form class ="" id="filter-form" action="" role="form">
+            <div class="col-lg-6 col-md-6 col-sm-6 col-xs-6" style="width: 65%;">
+                <a class="btn btn-block btn-primary" href="{% url 'website:questions' %}" style="margin-top: 18px;">View all previous questions.</a>
+            </div> <!-- /.col -->
+        </form>
+        <div class="col-lg-6 col-md-6 col-sm-6" style="width: 25%;">
+            <div class="row" style="margin-left: 11px;">
+
+                <div class="col-lg-6 col-md-6 col-sm-6 col-xs-6" style="width: 100px; margin-right:40px; " >
+                    <h3 style="font-size: 18px" align="center"><strong>{% total_question_count %}</strong></h3>
+                    <h3 style="font-size: 18px; margin-top: 1px;" align="center" text-color="#7395d9" >Questions</h3>
+                </div>
+
+            </div>
+        </div>
+    </div> <!-- /.row -->
 </div> <!-- /#filter-container -->
 
 <br>
 
 <div class="panel panel-default">
 
-	<div class="panel-heading">
-		Recent questions
-	</div>
-
-	<div class="panel-body">
-
-		<table id="myTable" class="tablesorter-blue"> 
-
-			<colgroup>
-				<col width="5%" />
-				<col width="20%" />
-				<col width="50%" />
-				<col width="8%" />
-				<col width="8%" />
-				<col width="8%" />
-			</colgroup>
-
-			<thead> 
-				<tr> 
-					<th>No.</th>
-					<th>Category</th>
-					<th>Question Title </th>
-					<th>Date</th>
-					<th>Views</th>
-					<th>Answers</th>
-				</tr> 
-			</thead>
-				
-			<tbody> 
-				{% for question in questions|get_recent_questions %}
-				<tr>
-					<td></td>
-					<td>
+    <div class="panel-heading">
+        Recent questions
+    </div>
+
+    <div class="panel-body">
+
+        <table id="myTable" class="tablesorter-blue">
+            Category : 
+            <select name="categories" id="category" style="margin: 10px 0px 15px 5px;">
+                <option>All Categories</option>
```

Comment:
Can not render the sub category as well as questions list for scilab-toolbox.

Reply:
solved